.github/workflows/test.sh

```diff
@@ -34,9 +34,21 @@ conda activate test-env-${BINDING}
 if [ "$USE_CONDA" = "No" ]; then
 
     if [ "${1}" = "pyqt5" ]; then
-        pip install pyqt5==${PYQT5_VERSION}.* PyQtWebEngine==${PYQT5_VERSION}.* QScintilla==${QSCINTILLA_VERSION}.*
+
+        if [ "$PYQT_EXTRAS" = "Yes" ]; then
+            pip install pyqt5==${PYQT5_VERSION}.* PyQtWebEngine==${PYQT5_VERSION}.* QScintilla==${QSCINTILLA_VERSION}.* PyQt3D==${PYQT5_VERSION}.* PyQtChart==${PYQT5_VERSION}.* PyQtDataVisualization==${PYQT5_VERSION}.* PyQtNetworkAuth==${PYQT5_VERSION}.* PyQtPurchasing==${PYQT5_VERSION}.*
```

Comment:
```suggestion
            pip install pyqt5==${PYQT5_VERSION}.* \
            PyQtWebEngine==${PYQT5_VERSION}.* \
            QScintilla==${QSCINTILLA_VERSION}.* \
            PyQt3D==${PYQT5_VERSION}.* \
            PyQtChart==${PYQT5_VERSION}.* \
            PyQtDataVisualization==${PYQT5_VERSION}.* \
            PyQtNetworkAuth==${PYQT5_VERSION}.* \
            PyQtPurchasing==${PYQT5_VERSION}.*
```

To make this easier to read.